test/contrib/test_ssltransport.py

```diff
@@ -408,6 +411,37 @@ def test_tls_in_tls_makefile_rw_binary(self):
                 validate_response(str_response, binary=False)
                 file.close()
 
+    @pytest.mark.timeout(PER_TEST_TIMEOUT)
+    def test_tls_in_tls_makefile_raw_rw_binary(self):
+        """
+        Uses makefile with read, write and binary modes without buffering.
+        """
+        self.start_destination_server()
+        self.start_proxy_server()
+
+        sock = socket.create_connection(
+            (self.proxy_server.host, self.proxy_server.port)
+        )
+        with self.client_context.wrap_socket(
+            sock, server_hostname="localhost"
+        ) as proxy_sock:
+            with SSLTransport(
+                proxy_sock, self.client_context, server_hostname="localhost"
+            ) as destination_sock:
+
+                file = destination_sock.makefile("rwb", buffering=0)
```

Comment:
Can you please use `pytest.mark.parametrize`? This `buffering=0` argument appears to be the only change from `test_tls_in_tls_makefile_rw_binary`. That way, if we fix one thing in one test it'll be fixed in the other too.